src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -25,7 +25,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         this.feedbackToUser = feedbackToUser;
         this.relevantPersons = relevantPersons;
     }
-
+    
+    public String getFeedbackToUser(){
+    	return feedbackToUser;
+    	
+    }
     /**
```

Comment:
changes looks good. Are you not using auto formatting? please hit auto-format after each change.